src/commonMain/kotlin/com.monta.changelog/git/GitService.kt

```diff
@@ -9,12 +9,14 @@ import kotlinx.datetime.toLocalDateTime
 
 class GitService(
     private val tagSorter: TagSorter,
-    tagPattern: String?
+    tagPattern: String?,
+    pathExcludePattern: String?
 ) {
 
     private val gitCommandUtil = GitCommandUtil()
     private val commitMapper = CommitMapper()
     private val tagPattern = if (tagPattern != null) Regex(tagPattern) else null
+    private val pathExcludePattern = if (pathExcludePattern != null) Regex(pathExcludePattern) else null
```

Comment:
nitpick (unimportant, it's a style thing): you could consider doing `xxx?.let { Regex(it) }` instead. 
```suggestion
    private val tagPattern = tagPattern?.let { Regex(it) }
    private val pathExcludePattern = pathExcludePattern?.let { Regex(it) }
```